omise/payment_methods/_all.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
I don't know how to put this, but I think it is kinda standard in a modern PHP already that we should have one class per file. This gives a bit of strange look. (I don't know how to explain 'why' it feels that way)

Reply:
This is very minor - I'll split it in the current branch I'm working on if it bothers you

Reply:
I've split it in my current branch